node/src/api/auth.cpp

```diff
@@ -17,11 +17,10 @@ const char COMPONENT_GROUP_NAME[] = "authorizations";
 
 }  // namespace
 
-auto auth(context_t& context, const std::string& name, const std::string& service) ->
-    std::shared_ptr<auth_t>
-{
+auto
+auth(context_t& context, const std::string& name) -> std::shared_ptr<auth_t> {
     if (auto cfg = context.config().component_group(COMPONENT_GROUP_NAME).get(name)) {
```

Comment:
AFAIR we use uppercase only for macro. Constants use the same style as variables, don't they?